judge/forms.py

```diff
@@ -284,3 +285,11 @@ def clean_key(self):
         if Contest.objects.filter(key=key).exists():
             raise ValidationError(_('Contest with key already exists.'))
         return key
+
+
+class ProblemPointsVoteForm(ModelForm):
+    note = CharField(max_length=8192, required=False)
```

Comment:
Is this too long, or too short? Leave this as a comment to the next reviewer (but feel free to comment your own opinions).

Reply:
for context, `Comment.body.max_length` is 8192, so i copied this number here

Reply:
I think 8192 is a reasonable limit for this and I doubt any non-malicious user would hit it. We definitely shouldn't go higher, but we could consider lowering the limit. It honestly does not matter too much though.

Reply:
This length seems fine to me.